usort/tests/cli.py

```diff
@@ -44,13 +44,12 @@ def test_benchmark(self) -> None:
             with chdir(dtmp):
                 result = runner.invoke(main, ["--benchmark", "check", "."])
 
-        self.assertRegex(
-            result.output,
-            r"""walking \.:\s+\d+ µs
-parsing sample\.py:\s+\d+ µs
-sorting sample\.py:\s+\d+ µs
-""",
-        )
+        for regex in (
```

Comment:
Is there an easy way to validate the order too?  I like that about the old version.

Reply:
Agreed, I meant to change this back to a static regex once I had it nailed down and forgot.